lighthouse-core/gather/gatherers/seo/tap-targets.js

```diff
@@ -5,16 +5,11 @@
  */
 'use strict';
 
-/* global document, window, getComputedStyle, getElementsInDocument, Node, getNodeDetails */
+/* global document, window, getComputedStyle, getElementsInDocument, Node, getNodeDetails, getRectCenterPoint */
 
 const Gatherer = require('../gatherer.js');
 const pageFunctions = require('../../../lib/page-functions.js');
-const {
-  rectContains,
-  getRectArea,
-  getRectCenterPoint,
-  getLargestRect,
-} = require('../../../lib/rect-helpers.js');
+const RectHelpers = require('../../../lib/rect-helpers.js');
```

Comment:
This is how all other gatherers import this module. Only this one did destructuring, which results in references in the "page functions" below to (lexically) check out. but lexical scope is a damned lie our tools tell themselves

Reply:
Ah, so I found this confusing, because obviously it has to be in scope to be able to `toString()` it, at which point it'll also be in scope on the browser side. The issue is the combo with [`keep_fnames: true`](https://github.com/GoogleChrome/lighthouse/blob/be88edd6bb8d50e4ce60a480fb0a533ec87822bb/build/build-bundle.js#L153-L154).

- Destructuring ends up looking something like `{getRectCenterPoint: o}  = e("../../../lib/rect-helpers.js")` since `getRectCenterPoint` wasn't renamed as a function, but as a local variable terser thinks it can be renamed.

- `o.toString` gives a function with the original name (`'function getRectCenterPoint() {...}'`), though, so when code tries to call `o()` it throws an error

Ideally we could eventually go the opposite way of this, having the `toString()` and the function calls match, as that would let us do `keep_fnames: false` and save 30KiB.